catalog/solutions/carboledger/2.0.0/solution.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "carboledger",
+    "name": "Carboledger",
+    "version": "2.0.0",
```

Comment:
You likely need 1.0.0 here too?

Reply:
The value of the `"version"` field here should be the same that is to be found on the file path (`catalog/solutions/<id>/<version>/solution.json`), so looking at this PR, I would say that `"2.0.0"` is the intended value here. If it should be `"1.0.0"`, please make sure that it's also changed in the path. 
Thank you!